src/test/java/io/confluent/connect/hdfs/wal/FSWALTest.java

```diff
@@ -72,4 +75,38 @@ public void testTruncatedVersionWalFileRecovery() throws Exception {
     FSWAL wal = new FSWAL("/logs", tp, storage);
     wal.acquireLease();
   }
+
+  @Test
+  public void testAcquireLeaseThrowsException() throws Exception {
```

Comment:
Based on your new fix, Do we need a new test that is testing the Writer instead of acquireLease.  We should leave this test, I mean this is a good test.

Reply:
Added a similar test - `io.confluent.connect.hdfs.wal.WALFileTest#testHdfsIsDown`.

Reply:
Thanks it look good